xrm/index.d.ts

```diff
@@ -394,11 +452,27 @@ declare namespace Xrm
     export interface Ui
     {
         /**
+          * Displays a form level notification. Any number of notifications can be displayed and will remain until removed using clearFormNotification.
+          * The height of the notification area is limited so each new message will be added to the top.
+          *
+          * @param   {string}                            message     The text of the notification message.
```

Comment:
I was just following the existing format that has been used for most of the other methods. Happy to remove them though.